.gitpod.yml

```diff
@@ -0,0 +1,18 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: yarn
+    init: yarn install
+    command: yarn run start
+# List the ports to expose. Learn more https://www.gitpod.io/docs/config-ports/
+ports:
```

Comment:
@Smartmind12 theres no need for open port

Reply:
Port 3000 is required so that the user can spin up the website or doc website on port 3000 and implement changes or testings on it.

Reply:
it depends on the future port we will decide to pick when we finish to migrate the docs from configu.com/docs to /docs dir in this repo (probably powered by https://docusaurus.io/ but we haven decided yet).